chonchon/rook-ceph/s3/rgw-rubintv.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: rubintv
+  namespace: rook-ceph
+spec:
+  metadataPool:
+    failureDomain: host
+    replicated:
+      size: 3
+    quotas:
+      maxSize: 10Gi
+  dataPool:
+    failureDomain: host
+    erasureCoded:
+      dataChunks: 2
+      codingChunks: 1
+    quotas:
+      maxSize: 5Ti
```

Comment:
Chonchon does not have the capacity to support 8Ti LFA + 5Ti rubintv.  The quota sizes should be reduced so we aren't overcommiting the storage.